console/src/components/Services/Events/Common/Components/InvocationLogsTable.tsx

```diff
@@ -100,11 +101,38 @@ const InvocationLogsTable: React.FC<Props> = props => {
     }
   };
 
-  const gridHeadings = sortedColumns.map(column => {
-    return {
+  const gridHeadings: any = [
+    {
```

Comment:
This is duplicated. Can you move it to some utility file and use both here and in `EventsTable.tsx`?